src/sdl2/hint.rs

```diff
@@ -3,6 +3,27 @@ use sys::hint as ll;
 use std::ptr;
 use libc::c_char;
 
+const VIDEO_MINIMIZE_ON_FOCUS_LOST: &'static str = "SDL_VIDEO_MINIMIZE_ON_FOCUS_LOST";
+
+/// Sets the SDL_HINT_VIDEO_MINIMIZE_ON_FOCUS_LOSS hint.
+///
+/// # Default
+/// By default a SDL_Window is minimized if it loses key focus when in fullscreen mode.
+///
+/// # Example
+/// ```no_run
+/// sdl2::hint::video_minimize_on_focus_lost(false);
+/// ```
+///
+/// * `value`:
+///    - `true`: do minimize the SDL_Window if it loses key focus when in fullscreen mode
+///    - `false`: do not minimize the SDL_Window if it loses key focus when in fullscreen mode
+///
+/// [Official SDL documentation](https://wiki.libsdl.org/SDL_HINT_VIDEO_MINIMIZE_ON_FOCUS_LOSS)
+pub fn video_minimize_on_focus_lost(value: bool) -> bool {
```

Comment:
As long as you can precisely which value does what, I'm fine with both. If there isn't any ambiguity like "I don't know if true or false actually enables the hint that I want to use because there is 'DISABLE' in the name" (I don't know if there is any actual hint like that, but just as an example). If there is no room for ambiguity, it's fine by me.